ts/indicators/acceleration-bands.src.ts

```diff
@@ -198,7 +198,7 @@ H.seriesType<Highcharts.ABandsIndicator>(
                 i: (number|undefined);
 
             if (yValLen < period) {
-                return false;
+                return undefined;
```

Comment:
Do not explicitly return `undefined` - just return without anything. Actually a lint rule should catch this. This has to be changed in all indicators.

Reply:
Ok, I'll fix, but why is returning nothing preferred?

Reply:
In some situations you can redeclare `undefined` as a variable. So use it only as a type and not as a value.